### DISCUSSION TITLE
Issue with Horizon theme: "TypeError: Cannot convert undefined or null to object"

### DISCUSSION THREAD

**Discussion by 0xSolidus** [swellstores/community#393] [Help]
I'm encountering a problem while trying to run the Horizon theme locally. When executing the command "yarn run dev", I come across a TypeError: Cannot convert undefined or null to object. Here is the error message:

$ graphql-codegen --config codegen.yml -r dotenv/config
✔ Parse Configuration
✔ Generate outputs
$ node -r dotenv/config ./build-utils/generate-theme.mjs
file:///home/mike/Desktop/horizon/build-utils/styles.mjs:14
  Object.entries(object).forEach(([key, value]) => {
         ^

TypeError: Cannot convert undefined or null to object
    at Function.entries (<anonymous>)
    at getVarsFromObject (file:///home/mike/Desktop/horizon/build-utils/styles.mjs:14:10)
    at generateCssVars (file:///home/mike/Desktop/horizon/build-utils/styles.mjs:32:21)
    at file:///home/mike/Desktop/horizon/build-utils/generate-theme.mjs:12:19
    at processTicksAndRejections (node:internal/process/task_queue:96:5)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

**Comment by errnerr** [comment]
@ARI-web23 - Did you come across a fix for this? I'm seeing the same error. I've messaged support but no fix yet.
